sydent/http/servlets/threepidunbindservlet.py

```diff
@@ -64,31 +67,80 @@ def _async_render_POST(self, request):
                 request.finish()
                 return
 
+            if 'sid' in body and 'client_secret' in body:
```

Comment:
Can you comment that there are two different cases going on here and what they are please?